INSTALL.md

```diff
@@ -29,7 +29,7 @@ To install the Enterprise Edition you need an eZ Enterprise subscription and hav
 Once you've got all the required PHP extensions installed, you can get eZ Platform Enterprise Edition up and running with the following commands:
 
 ``` bash
-composer create-project --keep-vcs ezsystems/ezplatform-demo-ee ezplatform-ee ^2
+composer create-project --keep-vcs ezsystems/ezplatform-ee-demo platform-ee-demo ^2
```

Comment:
Why did you change the folder name?

Currently ezplatform-demo uses ´ezplatform´ as folder name just like ezplatform repo.
ref: https://github.com/ezsystems/ezplatform-demo/blob/master/INSTALL.md

Reply:
@damianz5 suggested me to do so, I'm guessing it looks more consistent.

Reply:
that's right, as Andre said we can change it to `ezplatform`

Reply:
I guess it just should be reverted, so `ezplatform-ee`, or?

Reply:
> I guess it just should be reverted, so ezplatform-ee, or?

unless @SylvainGuittard prefers we always use same folder name as repo, then yes _(my point is merely about consistency across repos)_. But lets wait to hear his input.

Reply:
I don't really have a preference. We can definitely use `ezplatform` for more consistency between repos.

Reply:
I'm ok with that, simplifies things on merges.

If we rather wanted folder to be same as repo, then we could change the command so we don't type it twice.

Reply:
I changed the name to `ezplatform` as you suggested. Thank you for suggestions.